### PR TITLE
Issue 1828

### PR DESCRIPTION
**Pull request by 48klocs**
This is to tidy up #1828 and probably future questions about how weapon ratings work under the covers.

Additionally, I added links to the privacy and ratings markdown files to the settings options.

settings links

Look!  It's there in the settings menu.

item review link

Oh!  And also where you can turn it on in the item review tab.

### CODE REVIEW

File: src/scss/_style.scss [hunk 1]
```diff
@@ -1606,6 +1606,14 @@ h2, h3, h4 {
   color: #aaa;
 }
 
+.alongside {
+  float: right;
+}
```
Comment (bhollis) [inline src/scss/_style.scss]: This looks leftover.

**Comment by bhollis** [comment]
Wow, this is fantastic! If you clean up that one stray style I'll merge it straight away.

**Comment by 48klocs** [comment]
I'll double check to see if it's still needed - one of the question mark links was overflowing beneath the text without it earlier.

**Comment by 48klocs** [comment]
@bhollis fixed the stranded style.

Must have been an earlier point in time thing.  Good eye!

**Pull request merged**